src/components/PostComment.jsx

```diff
@@ -0,0 +1,73 @@
+import propTypes from "prop-types";
+import { UserContext } from "../contexts/User";
+import { useContext, useState } from "react";
+import { postData } from "./api";
+import styles from "../css/PostComment.module.css";
+
+function PostComment({ article_id }) {
+  const { user } = useContext(UserContext);
+  const [postedComment, setPostedComment] = useState({
+    author: user,
+    body: "",
+  });
+  const [isButtonDisabled, setButtonDisabled] = useState(false);
+  const [isSubmitted, setIsSubmitted] = useState(false);
+
+  function handleSubmit(event) {
+    setIsSubmitted(false);
```

Comment:
optimistically render the comment in the comments array so that the user can see the comment straight away without having to refresh the page